src/components/DropdownList.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../types/Person';
+
+type Props = {
+  people: Person[];
+  onClick?: (person: Person) => void;
+};
+
+export const DropdownList: React.FC<Props> = ({
+  people,
+  onClick = () => { },
```

Comment:
You can prevent extra renders by using `React.memo` here. (Also wrap `handleSelectPerson` in `useCallback`, as we are will passing to memo).